sdk/identity/Azure.Identity.BrokeredAuthentication/api/Azure.Identity.BrokeredAuthentication.net462.cs

```diff
@@ -2,11 +2,11 @@ namespace Azure.Identity.BrokeredAuthentication
 {
     public partial class InteractiveBrowserCredentialBrokerOptions : Azure.Identity.InteractiveBrowserCredentialOptions
     {
-        public InteractiveBrowserCredentialBrokerOptions(System.IntPtr parentWindowHandle) { }
+        public InteractiveBrowserCredentialBrokerOptions(System.IntPtr parentWindowHandle, bool msaPassthrough = false) { }
```

Comment:
Nit: Let's change this parameter name's casing all throughout for improved readability.
```suggestion
        public InteractiveBrowserCredentialBrokerOptions(System.IntPtr parentWindowHandle, bool msaPassThrough = false) { }
```